site/docs/contribute.md

```diff
@@ -145,6 +148,84 @@ Example:
   void sequenceNumber(long sequenceNumber);
 ```
 
+## Adding new functionality without breaking APIs
+Ideally, we'd want to add new functionality without breaking existing APIs, especially within the scope of the API modules that are being checked by [Revapi](https://revapi.org/).
```

Comment:
```suggestion
When adding new functionality, make sure to avoid breaking existing APIs,  especially within the scope of the API modules that are being checked by [Revapi](https://revapi.org/).
```